blocks/init/src/Blocks/wrapper/components/wrapper-toolbar.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { DropdownMenu, MenuGroup, MenuItem } from '@wordpress/components';
+import { icons } from '@eightshift/frontend-libs/scripts';
+
+export const WrapperToolbar = ({attributes, setAttributes}) => {
+
+    const copyAttributes = (closeMenu) => {
+        localStorage.removeItem('esCopiedWrapperAttributes');
+
+        const copiedWrapperAttributes = Object.keys(attributes).filter((key) => key.includes('wrapper'))
+        .reduce((cur, key) => { 
+            cur[key] = attributes[key];
+            return cur;
+        }, {});
+
+        localStorage.setItem('esCopiedWrapperAttributes', JSON.stringify(copiedWrapperAttributes));
+        closeMenu();
+    };
+
+    const pasteAttributes = (closeMenu) => {
+        const wrapperAttributesToBePasted = JSON.parse(localStorage.getItem('esCopiedWrapperAttributes'));
+        setAttributes(wrapperAttributesToBePasted);
+        closeMenu();
+    };
+
+	return (
+        <DropdownMenu 
+            icon={icons.wrapperConfig}
+            label={__('Copy/paste wrapper attributes', 'eighsfhit-frontend-libs')}
+            className="components-toolbar es-toolbar-icon-24"
+        >
+            { ( { onClose } ) => (
```

Comment:
This seems like a bit strange indent.

maybe something like
```suggestion
            { ({ onClose }) => (
```
or similar would look better. Just a suggestion so it doesn't really matter